com.hazelcast.jet.core.JobTest.when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission

_master_ (commit 6d892814f89bf4788bc30b79eb523b00e4706ed7)

Failed on Zing JDK 8: http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-ZingJDK8/310/testReport/com.hazelcast.jet.core/JobTest/when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission/

Stacktrace:
```
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.fail(Assert.java:96)
	at com.hazelcast.jet.core.JobTest.joinAndExpectCancellation(JobTest.java:978)
	at com.hazelcast.jet.core.JobTest.when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(JobTest.java:829)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

Standard output:
```
08:07:56,474  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-0ffe-0001 based on submit request
08:07:56,475 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-0ffe-0001 (name ??)
08:07:56,476  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-0fff-0001
08:07:56,476  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-0fff-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,476 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-0fff-0001
08:07:56,477 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-0fff-0001
08:07:56,477 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-0ffe-0001, execution 0660-ef95-0fff-0001 from [127.0.0.1]:5701
08:07:56,478  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-0ffe-0001, jobName='foo', executionId=0660-ef95-0fff-0001 initialized
08:07:56,478 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-0ffe-0001, execution 0660-ef95-0fff-0001 from [127.0.0.1]:5701
08:07:56,479  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-0ffe-0001, jobName='foo', executionId=0660-ef95-0fff-0001 initialized
08:07:56,479 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-0fff-0001 to be cancelled after Init
08:07:56,480 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-0fff-0001 calling completeExecution because execution terminated before it started
08:07:56,480 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-0fff-0001
08:07:56,481 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-0fff-0001 calling completeExecution because execution terminated before it started
08:07:56,481 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-0fff-0001
08:07:56,481  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-0fff-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.474
	Duration: 00:00:00.007
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,481 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-0fff-0001 is completed
08:07:56,483  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-1000-0001 based on submit request
08:07:56,484 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-1000-0001 (name ??)
08:07:56,484  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-1001-0001
08:07:56,484  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-1001-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,484 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-1001-0001
08:07:56,485 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-1001-0001
08:07:56,485 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1000-0001, execution 0660-ef95-1001-0001 from [127.0.0.1]:5701
08:07:56,485  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1000-0001, jobName='foo', executionId=0660-ef95-1001-0001 initialized
08:07:56,486 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1000-0001, execution 0660-ef95-1001-0001 from [127.0.0.1]:5701
08:07:56,487  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1000-0001, jobName='foo', executionId=0660-ef95-1001-0001 initialized
08:07:56,487 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1001-0001 to be cancelled after Init
08:07:56,487 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-4 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1001-0001 calling completeExecution because execution terminated before it started
08:07:56,487 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-4 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1001-0001
08:07:56,487 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1001-0001 calling completeExecution because execution terminated before it started
08:07:56,487 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1001-0001
08:07:56,488  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-1001-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.483
	Duration: 00:00:00.005
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,488 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1001-0001 is completed
08:07:56,490  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-1002-0001 based on submit request
08:07:56,490 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-1002-0001 (name ??)
08:07:56,491  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-1003-0001
08:07:56,491  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-1003-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,491 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-1003-0001
08:07:56,491 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-1003-0001
08:07:56,492 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1002-0001, execution 0660-ef95-1003-0001 from [127.0.0.1]:5701
08:07:56,492  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1002-0001, jobName='foo', executionId=0660-ef95-1003-0001 initialized
08:07:56,493 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1002-0001, execution 0660-ef95-1003-0001 from [127.0.0.1]:5701
08:07:56,493  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1002-0001, jobName='foo', executionId=0660-ef95-1003-0001 initialized
08:07:56,494 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1003-0001 to be cancelled after Init
08:07:56,494 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1003-0001 calling completeExecution because execution terminated before it started
08:07:56,494 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1003-0001
08:07:56,494 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1003-0001 calling completeExecution because execution terminated before it started
08:07:56,494 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1003-0001
08:07:56,494  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-1003-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.490
	Duration: 00:00:00.004
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,495 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1003-0001 is completed
08:07:56,497  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-1004-0001 based on submit request
08:07:56,497 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-1004-0001 (name ??)
08:07:56,499  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-1005-0001
08:07:56,499  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-1005-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,499 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-1005-0001
08:07:56,499 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-1005-0001
08:07:56,499 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1004-0001, execution 0660-ef95-1005-0001 from [127.0.0.1]:5701
08:07:56,500  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1004-0001, jobName='foo', executionId=0660-ef95-1005-0001 initialized
08:07:56,500 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1004-0001, execution 0660-ef95-1005-0001 from [127.0.0.1]:5701
08:07:56,501  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1004-0001, jobName='foo', executionId=0660-ef95-1005-0001 initialized
08:07:56,501 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1005-0001 to be cancelled after Init
08:07:56,501 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1005-0001 calling completeExecution because execution terminated before it started
08:07:56,501 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1005-0001
08:07:56,501 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1005-0001 calling completeExecution because execution terminated before it started
08:07:56,501 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1005-0001
08:07:56,502  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-1005-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.497
	Duration: 00:00:00.005
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,502 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1005-0001 is completed
08:07:56,504  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-1006-0001 based on submit request
08:07:56,505 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-1006-0001 (name ??)
08:07:56,505  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-1007-0001
08:07:56,505  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-1007-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,506 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-1007-0001
08:07:56,506 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-1007-0001
08:07:56,506 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1006-0001, execution 0660-ef95-1007-0001 from [127.0.0.1]:5701
08:07:56,506  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1006-0001, jobName='foo', executionId=0660-ef95-1007-0001 initialized
08:07:56,507 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1006-0001, execution 0660-ef95-1007-0001 from [127.0.0.1]:5701
08:07:56,508  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1006-0001, jobName='foo', executionId=0660-ef95-1007-0001 initialized
08:07:56,508 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1007-0001 to be cancelled after Init
08:07:56,509 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1007-0001 calling completeExecution because execution terminated before it started
08:07:56,509 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1007-0001
08:07:56,512 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1007-0001 calling completeExecution because execution terminated before it started
08:07:56,512 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1007-0001
08:07:56,516  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-1007-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.504
	Duration: 00:00:00.012
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,517 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1007-0001 is completed
08:07:56,519  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-1008-0001 based on submit request
08:07:56,520 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-1008-0001 (name ??)
08:07:56,521  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-1009-0001
08:07:56,521  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-1009-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,521 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-1009-0001
08:07:56,521 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-1009-0001
08:07:56,522 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1008-0001, execution 0660-ef95-1009-0001 from [127.0.0.1]:5701
08:07:56,522  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1008-0001, jobName='foo', executionId=0660-ef95-1009-0001 initialized
08:07:56,523 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-1008-0001, execution 0660-ef95-1009-0001 from [127.0.0.1]:5701
08:07:56,524  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-1008-0001, jobName='foo', executionId=0660-ef95-1009-0001 initialized
08:07:56,524 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1009-0001 to be cancelled after Init
08:07:56,524 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-4 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1009-0001 calling completeExecution because execution terminated before it started
08:07:56,524 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-4 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1009-0001
08:07:56,524 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1009-0001 calling completeExecution because execution terminated before it started
08:07:56,524 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-1009-0001
08:07:56,525  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-1009-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.519
	Duration: 00:00:00.005
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,526 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-1009-0001 is completed
08:07:56,527  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-100a-0001 based on submit request
08:07:56,527 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-100a-0001 (name ??)
08:07:56,529  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-100b-0001
08:07:56,529  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-100b-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,529 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-100b-0001
08:07:56,529 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-100b-0001
08:07:56,529 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100a-0001, execution 0660-ef95-100b-0001 from [127.0.0.1]:5701
08:07:56,530  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100a-0001, jobName='foo', executionId=0660-ef95-100b-0001 initialized
08:07:56,530 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100a-0001, execution 0660-ef95-100b-0001 from [127.0.0.1]:5701
08:07:56,531  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100a-0001, jobName='foo', executionId=0660-ef95-100b-0001 initialized
08:07:56,531 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100b-0001 to be cancelled after Init
08:07:56,531 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100b-0001 calling completeExecution because execution terminated before it started
08:07:56,531 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100b-0001
08:07:56,531 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100b-0001 calling completeExecution because execution terminated before it started
08:07:56,532 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100b-0001
08:07:56,532  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100b-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.527
	Duration: 00:00:00.005
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,532 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100b-0001 is completed
08:07:56,534  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-100c-0001 based on submit request
08:07:56,534 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-100c-0001 (name ??)
08:07:56,536  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-100d-0001
08:07:56,536  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-100d-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,536 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-100d-0001
08:07:56,536 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-100d-0001
08:07:56,536 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100c-0001, execution 0660-ef95-100d-0001 from [127.0.0.1]:5701
08:07:56,537  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100c-0001, jobName='foo', executionId=0660-ef95-100d-0001 initialized
08:07:56,537 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100c-0001, execution 0660-ef95-100d-0001 from [127.0.0.1]:5701
08:07:56,538  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100c-0001, jobName='foo', executionId=0660-ef95-100d-0001 initialized
08:07:56,538 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100d-0001 to be cancelled after Init
08:07:56,538 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100d-0001 calling completeExecution because execution terminated before it started
08:07:56,538 DEBUG || - [JobExecutionService] hz.naughty_franklin.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100d-0001
08:07:56,538 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100d-0001 calling completeExecution because execution terminated before it started
08:07:56,538 DEBUG || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100d-0001
08:07:56,539  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100d-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-06-22T08:07:56.534
	Duration: 00:00:00.005
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,540 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100d-0001 is completed
08:07:56,542  INFO || - [JobCoordinationService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 0660-ef95-100e-0001 based on submit request
08:07:56,544  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 0660-ef95-100f-0001
08:07:56,544  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 0660-ef95-100f-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
08:07:56,544 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 0660-ef95-100f-0001
08:07:56,544 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 0660-ef95-100f-0001
08:07:56,545 DEBUG || - [InitExecutionOperation] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100e-0001, execution 0660-ef95-100f-0001 from [127.0.0.1]:5701
08:07:56,545  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100e-0001, jobName='foo', executionId=0660-ef95-100f-0001 initialized
08:07:56,546 DEBUG || - [InitExecutionOperation] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 0660-ef95-100e-0001, execution 0660-ef95-100f-0001 from [127.0.0.1]:5701
08:07:56,546  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=0660-ef95-100e-0001, jobName='foo', executionId=0660-ef95-100f-0001 initialized
08:07:56,547 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Init of job 'foo', execution 0660-ef95-100f-0001 was successful
08:07:56,547 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Executing job 'foo', execution 0660-ef95-100f-0001
08:07:56,547  INFO || - [JobExecutionService] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start execution of job 'foo', execution 0660-ef95-100f-0001 from coordinator [127.0.0.1]:5701
08:07:56,547  INFO || - [JobExecutionService] hz.charming_franklin.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Start execution of job 'foo', execution 0660-ef95-100f-0001 from coordinator [127.0.0.1]:5701
08:07:56,548 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100f-0001
08:07:56,548 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100f-0001 completed
08:07:56,548 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100f-0001 received response to StartExecutionOperation from [127.0.0.1]:5701: null
08:07:56,551 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-214 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 0660-ef95-100f-0001
08:07:56,551 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-214 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100f-0001 completed
08:07:56,551 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-214 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100f-0001 received response to StartExecutionOperation from [127.0.0.1]:5702: null
08:07:56,551 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-214 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100f-0001 was successful
08:07:56,551 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 0660-ef95-100e-0001 (name ??)
08:07:56,551  INFO || - [MasterJobContext] hz.naughty_franklin.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 0660-ef95-100f-0001 completed successfully
	Start time: 2021-06-22T08:07:56.542
	Duration: 00:00:00.009
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
08:07:56,552 DEBUG || - [MasterJobContext] hz.naughty_franklin.cached.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Not scaling up job 'foo', execution 0660-ef95-100f-0001: not running or already running on all members
08:07:56,552 DEBUG || - [JobCoordinationService] hz.naughty_franklin.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 0660-ef95-100f-0001 is completed
08:07:56,578  INFO |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [SimpleTestInClusterSupport] Time-limited test - Ditching 77 jobs in SimpleTestInClusterSupport.@After: [0660-ef95-0ff0-0001, 0660-ef95-0ff2-0001, 0660-ef95-0ffe-0001, 0660-ef95-0fe4-0001, 0660-ef91-2362-0001, 0660-ef95-0fe6-0001, 0660-ef95-0fe0-0001, 0660-ef95-0fe2-0001, 0660-ef95-0fee-0001, 0660-ef95-0fe8-0001, 0660-ef95-0fea-0001, 0660-ef91-2350-0001, 0660-ef95-0fd5-0001, 0660-ef95-0fd7-0001, 0660-ef91-2355-0001, 0660-ef95-0fdc-0001, 0660-ef95-0fde-0001, 0660-ef91-235c-0001, 0660-ef95-0fda-0001, 0660-ef91-2341-0001, 0660-ef94-a542-0002, 0660-ef94-a540-0002, 0660-ef91-2346-0001, 0660-ef95-0fc3-0001, 0660-ef95-0fcf-0001, 0660-ef91-234c-0001, 0660-ef95-0fc9-0001, 0660-ef91-2330-0001, 0660-ef91-3fb2-0002, 0660-ef91-2332-0001, 0660-ef91-2334-0001, 0660-ef91-2337-0001, 0660-ef91-233a-0001, 0660-ef91-233c-0001, 0660-ef91-233e-0001, 0660-ef91-2321-0001, 0660-ef91-2323-0001, 0660-ef91-2325-0001, 0660-ef91-2327-0001, 0660-ef91-3fa4-0002, 0660-ef91-2329-0001, 0660-ef91-232b-0001, 0660-ef91-3fa8-0002, 0660-ef91-232d-0001, 0660-ef91-3fad-0002, 0660-ef91-2310-0001, 0660-ef91-2312-0001, 0660-ef91-3f90-0002, 0660-ef91-2315-0001, 0660-ef91-2317-0001, 0660-ef91-3f94-0002, 0660-ef91-2319-0001, 0660-ef91-3f9a-0002, 0660-ef91-231b-0001, 0660-ef91-3f9e-0002, 0660-ef91-231d-0001, 0660-ef91-231f-0001, 0660-ef95-1004-0001, 0660-ef91-3f82-0002, 0660-ef95-1006-0001, 0660-ef91-3f81-0002, 0660-ef91-2302-0001, 0660-ef91-3f80-0002, 0660-ef95-1000-0001, 0660-ef91-3f87-0002, 0660-ef91-2304-0001, 0660-ef95-1002-0001, 0660-ef91-2307-0001, 0660-ef91-3f84-0002, 0660-ef95-100c-0001, 0660-ef91-3f8a-0002, 0660-ef91-230a-0001, 0660-ef95-100e-0001, 0660-ef95-1008-0001, 0660-ef91-230c-0001, 0660-ef95-100a-0001, 0660-ef91-230e-0001]
08:07:56,602  INFO |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [SimpleTestInClusterSupport] Time-limited test - Destroying 0 distributed objects in SimpleTestInClusterSupport.@After: []
BuildInfo right after when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(com.hazelcast.jet.core.JobTest): BuildInfo{version='5.0-SNAPSHOT', build='20210622', buildNumber=20210622, revision=6d89281, enterprise=false, serializationVersion=1, jet=JetBuildInfo{version='5.0-SNAPSHOT', build='20210622', revision='6d89281'}}
Hiccups measured while running test 'when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(com.hazelcast.jet.core.JobTest):'
08:07:55, accumulated pauses: 1573 ms, max pause: 12 ms, pauses over 1000 ms: 0
```

Reopening. The test failed on `master` branch. (Mac OS)

### Stacktrace
http://jenkins.hazelcast.com/job/Hazelcast-master-Macos-OpenJDK8/2/com.hazelcast$hazelcast/testReport/com.hazelcast.jet.core/JobTest/when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission/

```
java.lang.AssertionError: 

Actual and expected have the same elements but not in the same order, at index 6 actual element was:
  <487320366702526465L>
whereas expected element was:
  <487320366702592001L>

	at com.hazelcast.jet.core.JobTest.when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(JobTest.java:850)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

### Standard Output

```
Finished Running Test: when_namedJobIsRunning_then_newNamedJobFails_client in 0.211 seconds.
Started Running Test: when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission
17:57:15,272  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a51b-0001 based on submit request
17:57:15,272 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a51b-0001 (name ??)
17:57:15,272  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6db-0001
17:57:15,272  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6db-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,272 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6db-0001
17:57:15,272 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6db-0001
17:57:15,272 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51b-0001, execution 06c3-4f4a-a6db-0001 from [127.0.0.1]:5701
17:57:15,272 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51b-0001, execution 06c3-4f4a-a6db-0001 from [127.0.0.1]:5701
17:57:15,272  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51b-0001, jobName='foo', executionId=06c3-4f4a-a6db-0001 initialized
17:57:15,272  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51b-0001, jobName='foo', executionId=06c3-4f4a-a6db-0001 initialized
17:57:15,272 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6db-0001 to be cancelled after Init
17:57:15,272 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6db-0001 calling completeExecution because execution terminated before it started
17:57:15,272 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6db-0001 calling completeExecution because execution terminated before it started
17:57:15,272 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6db-0001
17:57:15,272 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6db-0001
17:57:15,272  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6db-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.272
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,273 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6db-0001 is completed
17:57:15,273  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a51c-0001 based on submit request
17:57:15,273 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a51c-0001 (name ??)
17:57:15,273  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6dc-0001
17:57:15,273  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6dc-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,273 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6dc-0001
17:57:15,273 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6dc-0001
17:57:15,273 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51c-0001, execution 06c3-4f4a-a6dc-0001 from [127.0.0.1]:5701
17:57:15,273 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51c-0001, execution 06c3-4f4a-a6dc-0001 from [127.0.0.1]:5701
17:57:15,273  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51c-0001, jobName='foo', executionId=06c3-4f4a-a6dc-0001 initialized
17:57:15,273  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51c-0001, jobName='foo', executionId=06c3-4f4a-a6dc-0001 initialized
17:57:15,273 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dc-0001 to be cancelled after Init
17:57:15,273 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dc-0001 calling completeExecution because execution terminated before it started
17:57:15,273 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dc-0001 calling completeExecution because execution terminated before it started
17:57:15,273 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-7 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6dc-0001
17:57:15,273 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6dc-0001
17:57:15,273  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6dc-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.273
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,274 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dc-0001 is completed
17:57:15,274  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a51d-0001 based on submit request
17:57:15,274 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a51d-0001 (name ??)
17:57:15,274  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6dd-0001
17:57:15,274  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6dd-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,274 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6dd-0001
17:57:15,274 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6dd-0001
17:57:15,274 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51d-0001, execution 06c3-4f4a-a6dd-0001 from [127.0.0.1]:5701
17:57:15,274 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51d-0001, execution 06c3-4f4a-a6dd-0001 from [127.0.0.1]:5701
17:57:15,274  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51d-0001, jobName='foo', executionId=06c3-4f4a-a6dd-0001 initialized
17:57:15,274  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51d-0001, jobName='foo', executionId=06c3-4f4a-a6dd-0001 initialized
17:57:15,274 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dd-0001 to be cancelled after Init
17:57:15,274 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dd-0001 calling completeExecution because execution terminated before it started
17:57:15,274 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dd-0001 calling completeExecution because execution terminated before it started
17:57:15,274 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6dd-0001
17:57:15,274 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6dd-0001
17:57:15,274  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6dd-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.274
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,274 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6dd-0001 is completed
17:57:15,275  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a51e-0001 based on submit request
17:57:15,275 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a51e-0001 (name ??)
17:57:15,275  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6de-0001
17:57:15,275  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6de-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,275 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6de-0001
17:57:15,275 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6de-0001
17:57:15,275 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51e-0001, execution 06c3-4f4a-a6de-0001 from [127.0.0.1]:5701
17:57:15,275 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51e-0001, execution 06c3-4f4a-a6de-0001 from [127.0.0.1]:5701
17:57:15,275  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51e-0001, jobName='foo', executionId=06c3-4f4a-a6de-0001 initialized
17:57:15,275  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51e-0001, jobName='foo', executionId=06c3-4f4a-a6de-0001 initialized
17:57:15,275 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6de-0001 to be cancelled after Init
17:57:15,275 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6de-0001 calling completeExecution because execution terminated before it started
17:57:15,275 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-9 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6de-0001 calling completeExecution because execution terminated before it started
17:57:15,275 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6de-0001
17:57:15,275 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-9 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6de-0001
17:57:15,275  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6de-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.275
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,275 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6de-0001 is completed
17:57:15,276  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a51f-0001 based on submit request
17:57:15,276 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a51f-0001 (name ??)
17:57:15,276  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6df-0001
17:57:15,276  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6df-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,276 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6df-0001
17:57:15,276 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6df-0001
17:57:15,276 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51f-0001, execution 06c3-4f4a-a6df-0001 from [127.0.0.1]:5701
17:57:15,276 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a51f-0001, execution 06c3-4f4a-a6df-0001 from [127.0.0.1]:5701
17:57:15,276  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51f-0001, jobName='foo', executionId=06c3-4f4a-a6df-0001 initialized
17:57:15,276  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a51f-0001, jobName='foo', executionId=06c3-4f4a-a6df-0001 initialized
17:57:15,276 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6df-0001 to be cancelled after Init
17:57:15,276 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6df-0001 calling completeExecution because execution terminated before it started
17:57:15,276 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6df-0001 calling completeExecution because execution terminated before it started
17:57:15,276 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6df-0001
17:57:15,276 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6df-0001
17:57:15,276  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6df-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.276
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,276 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6df-0001 is completed
17:57:15,276  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a520-0001 based on submit request
17:57:15,276 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a520-0001 (name ??)
17:57:15,277  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e0-0001
17:57:15,277  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e0-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,277 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e0-0001
17:57:15,277 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e0-0001
17:57:15,277 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a520-0001, execution 06c3-4f4a-a6e0-0001 from [127.0.0.1]:5701
17:57:15,277 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a520-0001, execution 06c3-4f4a-a6e0-0001 from [127.0.0.1]:5701
17:57:15,277  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a520-0001, jobName='foo', executionId=06c3-4f4a-a6e0-0001 initialized
17:57:15,277  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a520-0001, jobName='foo', executionId=06c3-4f4a-a6e0-0001 initialized
17:57:15,277 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e0-0001 to be cancelled after Init
17:57:15,277 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-11 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e0-0001 calling completeExecution because execution terminated before it started
17:57:15,277 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e0-0001 calling completeExecution because execution terminated before it started
17:57:15,277 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-11 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e0-0001
17:57:15,277 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e0-0001
17:57:15,277  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e0-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.276
	Duration: 00:00:00.001
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,277 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e0-0001 is completed
17:57:15,277  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a521-0001 based on submit request
17:57:15,277 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a521-0001 (name ??)
17:57:15,277  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e1-0001
17:57:15,277  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e1-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,277 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e1-0001
17:57:15,277 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e1-0001
17:57:15,277 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a521-0001, execution 06c3-4f4a-a6e1-0001 from [127.0.0.1]:5701
17:57:15,277 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a521-0001, execution 06c3-4f4a-a6e1-0001 from [127.0.0.1]:5701
17:57:15,278  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a521-0001, jobName='foo', executionId=06c3-4f4a-a6e1-0001 initialized
17:57:15,278  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a521-0001, jobName='foo', executionId=06c3-4f4a-a6e1-0001 initialized
17:57:15,278 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e1-0001 to be cancelled after Init
17:57:15,278 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e1-0001 calling completeExecution because execution terminated before it started
17:57:15,278 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e1-0001 calling completeExecution because execution terminated before it started
17:57:15,278 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-12 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e1-0001
17:57:15,278 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e1-0001
17:57:15,278  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e1-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.277
	Duration: 00:00:00.001
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,278 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e1-0001 is completed
17:57:15,278  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a522-0001 based on submit request
17:57:15,278 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a522-0001 (name ??)
17:57:15,278  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e2-0001
17:57:15,278  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e2-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,278 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e2-0001
17:57:15,278 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e2-0001
17:57:15,278 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a522-0001, execution 06c3-4f4a-a6e2-0001 from [127.0.0.1]:5701
17:57:15,278 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a522-0001, execution 06c3-4f4a-a6e2-0001 from [127.0.0.1]:5701
17:57:15,278  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a522-0001, jobName='foo', executionId=06c3-4f4a-a6e2-0001 initialized
17:57:15,278  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a522-0001, jobName='foo', executionId=06c3-4f4a-a6e2-0001 initialized
17:57:15,279 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e2-0001 to be cancelled after Init
17:57:15,279 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e2-0001 calling completeExecution because execution terminated before it started
17:57:15,279 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e2-0001 calling completeExecution because execution terminated before it started
17:57:15,279 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-1 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e2-0001
17:57:15,279 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e2-0001
17:57:15,279  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e2-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.278
	Duration: 00:00:00.001
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,279 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e2-0001 is completed
17:57:15,279  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a523-0001 based on submit request
17:57:15,279 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a523-0001 (name ??)
17:57:15,279  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e3-0001
17:57:15,279  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e3-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,279 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e3-0001
17:57:15,279 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e3-0001
17:57:15,279 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a523-0001, execution 06c3-4f4a-a6e3-0001 from [127.0.0.1]:5701
17:57:15,279 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a523-0001, execution 06c3-4f4a-a6e3-0001 from [127.0.0.1]:5701
17:57:15,279  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a523-0001, jobName='foo', executionId=06c3-4f4a-a6e3-0001 initialized
17:57:15,279  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a523-0001, jobName='foo', executionId=06c3-4f4a-a6e3-0001 initialized
17:57:15,279 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e3-0001 to be cancelled after Init
17:57:15,279 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e3-0001 calling completeExecution because execution terminated before it started
17:57:15,279 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e3-0001 calling completeExecution because execution terminated before it started
17:57:15,279 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e3-0001
17:57:15,279 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e3-0001
17:57:15,279  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e3-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.279
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,280 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e3-0001 is completed
17:57:15,280  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a524-0001 based on submit request
17:57:15,280 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [Job] when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a524-0001 (name ??)
17:57:15,280  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e4-0001
17:57:15,280  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e4-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,280 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e4-0001
17:57:15,280 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e4-0001
17:57:15,280 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a524-0001, execution 06c3-4f4a-a6e4-0001 from [127.0.0.1]:5701
17:57:15,280 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a524-0001, execution 06c3-4f4a-a6e4-0001 from [127.0.0.1]:5701
17:57:15,280  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a524-0001, jobName='foo', executionId=06c3-4f4a-a6e4-0001 initialized
17:57:15,280  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a524-0001, jobName='foo', executionId=06c3-4f4a-a6e4-0001 initialized
17:57:15,280 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e4-0001 to be cancelled after Init
17:57:15,280 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e4-0001 calling completeExecution because execution terminated before it started
17:57:15,280 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e4-0001 calling completeExecution because execution terminated before it started
17:57:15,280 DEBUG || - [JobExecutionService] hz.dreamy_turing.async.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e4-0001
17:57:15,280 DEBUG || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e4-0001
17:57:15,280  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e4-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.280
	Duration: 00:00:00.000
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,280 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e4-0001 is completed
17:57:15,281  INFO || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Starting job 06c3-4f4a-a525-0001 based on submit request
17:57:15,281  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Didn't find any snapshot to restore for job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,281  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start executing job 'foo', execution 06c3-4f4a-a6e5-0001, execution graph in DOT format:
digraph DAG {
	"v" [localParallelism=1];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
17:57:15,281 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Building execution plan for job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,281 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Built execution plans for job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,281 DEBUG || - [InitExecutionOperation] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a525-0001, execution 06c3-4f4a-a6e5-0001 from [127.0.0.1]:5701
17:57:15,281 DEBUG || - [InitExecutionOperation] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Initializing execution plan for job 06c3-4f4a-a525-0001, execution 06c3-4f4a-a6e5-0001 from [127.0.0.1]:5701
17:57:15,281  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a525-0001, jobName='foo', executionId=06c3-4f4a-a6e5-0001 initialized
17:57:15,281  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution plan for jobId=06c3-4f4a-a525-0001, jobName='foo', executionId=06c3-4f4a-a6e5-0001 initialized
17:57:15,281 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Init of job 'foo', execution 06c3-4f4a-a6e5-0001 was successful
17:57:15,281 DEBUG || - [MasterJobContext] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Executing job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,281  INFO || - [JobExecutionService] hz.dreamy_turing.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Start execution of job 'foo', execution 06c3-4f4a-a6e5-0001 from coordinator [127.0.0.1]:5701
17:57:15,281  INFO || - [JobExecutionService] hz.peaceful_turing.generic-operation.thread-1 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Start execution of job 'foo', execution 06c3-4f4a-a6e5-0001 from coordinator [127.0.0.1]:5701
17:57:15,424  INFO |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [SimpleTestInClusterSupport] Time-limited test - Ditching 37 jobs in SimpleTestInClusterSupport.@After: [06c3-4f4a-a508-0001, 06c3-4f4a-a509-0001, 06c3-4f4a-a50a-0001, 06c3-4f4a-a50b-0001, 06c3-4f4a-a50c-0001, 06c3-4f4a-a50d-0001, 06c3-4f4a-a50e-0001, 06c3-4f4a-a50f-0001, 06c3-4f4b-ee42-0002, 06c3-4f4a-a502-0001, 06c3-4f4a-a504-0001, 06c3-4f4a-a505-0001, 06c3-4f4a-a506-0001, 06c3-4f4a-a507-0001, 06c3-4f4a-a518-0001, 06c3-4f4a-a519-0001, 06c3-4f50-2d00-0002, 06c3-4f4a-a51a-0001, 06c3-4f4a-a51b-0001, 06c3-4f4a-a51c-0001, 06c3-4f4a-a51d-0001, 06c3-4f4a-a51e-0001, 06c3-4f4a-a51f-0001, 06c3-4f4a-a510-0001, 06c3-4f4a-a511-0001, 06c3-4f4a-a512-0001, 06c3-4f4a-a513-0001, 06c3-4f4a-a514-0001, 06c3-4f4a-a515-0001, 06c3-4f4a-a516-0001, 06c3-4f4a-a517-0001, 06c3-4f4a-a520-0001, 06c3-4f4a-a521-0001, 06c3-4f4a-a522-0001, 06c3-4f4a-a523-0001, 06c3-4f4a-a524-0001, 06c3-4f4a-a525-0001]
17:57:15,426 DEBUG |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [AbstractJobProxy] Time-limited test - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Sending CANCEL_FORCEFUL request for job 06c3-4f4a-a525-0001 (name ??)
17:57:15,430 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,430 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-2 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Completed execution of job 'foo', execution 06c3-4f4a-a6e5-0001
17:57:15,430 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-2 - [127.0.0.1]:5702 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e5-0001 completed with failure
java.util.concurrent.CompletionException: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:783) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:750) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990) ~[?:1.8.0_302]
	at com.hazelcast.jet.impl.util.NonCompletableFuture.internalCompleteExceptionally(NonCompletableFuture.java:72) ~[classes/:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$ExecutionTracker.taskletDone(TaskletExecutionService.java:489) ~[classes/:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$BlockingWorker.run(TaskletExecutionService.java:317) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_302]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_302]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_302]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_302]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_302]
Caused by: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
	at com.hazelcast.jet.impl.execution.ExecutionContext.terminateExecution(ExecutionContext.java:294) ~[classes/:?]
	at com.hazelcast.jet.impl.JobExecutionService.terminateExecution0(JobExecutionService.java:711) ~[classes/:?]
	at com.hazelcast.jet.impl.JobExecutionService.terminateExecution(JobExecutionService.java:707) ~[classes/:?]
	at com.hazelcast.jet.impl.operation.TerminateExecutionOperation.run(TerminateExecutionOperation.java:58) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.Operation.call(Operation.java:196) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:273) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:249) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:470) ~[classes/:?]
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:197) ~[classes/:?]
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:137) ~[classes/:?]
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.executeRun(OperationThread.java:123) ~[classes/:?]
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102) ~[classes/:?]
17:57:15,430 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e5-0001 completed with failure
java.util.concurrent.CompletionException: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:783) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:750) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488) ~[?:1.8.0_302]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990) ~[?:1.8.0_302]
	at com.hazelcast.jet.impl.util.NonCompletableFuture.internalCompleteExceptionally(NonCompletableFuture.java:72) ~[classes/:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$ExecutionTracker.taskletDone(TaskletExecutionService.java:489) ~[classes/:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$BlockingWorker.run(TaskletExecutionService.java:317) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_302]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_302]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_302]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_302]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_302]
Caused by: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
	at com.hazelcast.jet.impl.execution.ExecutionContext.terminateExecution(ExecutionContext.java:294) ~[classes/:?]
	at com.hazelcast.jet.impl.JobExecutionService.terminateExecution0(JobExecutionService.java:711) ~[classes/:?]
	at com.hazelcast.jet.impl.JobExecutionService.terminateExecution(JobExecutionService.java:707) ~[classes/:?]
	at com.hazelcast.jet.impl.operation.TerminateExecutionOperation.run(TerminateExecutionOperation.java:58) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.Operation.call(Operation.java:196) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:273) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:249) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:214) ~[classes/:?]
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationExecutorImpl.run(OperationExecutorImpl.java:411) ~[classes/:?]
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationExecutorImpl.runOrExecute(OperationExecutorImpl.java:438) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvokeLocal(Invocation.java:600) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:579) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke0(Invocation.java:540) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke(Invocation.java:240) ~[classes/:?]
	at com.hazelcast.spi.impl.operationservice.impl.InvocationBuilderImpl.invoke(InvocationBuilderImpl.java:59) ~[classes/:?]
	at com.hazelcast.jet.impl.MasterContext.invokeOnParticipant(MasterContext.java:294) ~[classes/:?]
	at com.hazelcast.jet.impl.MasterContext.invokeOnParticipants(MasterContext.java:277) ~[classes/:?]
	at com.hazelcast.jet.impl.MasterJobContext.lambda$cancelExecutionInvocations$17(MasterJobContext.java:610) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_302]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_302]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_302]
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:76) ~[classes/:?]
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102) ~[classes/:?]
17:57:15,431 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-8 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e5-0001 received response to StartExecutionOperation from [127.0.0.1]:5701: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
17:57:15,431 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e5-0001 received response to StartExecutionOperation from [127.0.0.1]:5702: com.hazelcast.jet.impl.exception.JobTerminateRequestedException: CANCEL_FORCEFUL
17:57:15,431 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-2 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e5-0001 to be cancelled after Execution
17:57:15,431  INFO || - [MasterJobContext] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] Execution of job 'foo', execution 06c3-4f4a-a6e5-0001 got terminated, reason=java.util.concurrent.CancellationException
	Start time: 2021-09-06T17:57:15.281
	Duration: 00:00:00.150
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
17:57:15,431 DEBUG || - [JobCoordinationService] hz.dreamy_turing.cached.thread-10 - [127.0.0.1]:5701 [dev] [5.0-SNAPSHOT] job 'foo', execution 06c3-4f4a-a6e5-0001 is completed
17:57:15,431  INFO |when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission| - [SimpleTestInClusterSupport] Time-limited test - Destroying 0 distributed objects in SimpleTestInClusterSupport.@After: []
BuildInfo right after when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(com.hazelcast.jet.core.JobTest): BuildInfo{version='5.0-SNAPSHOT', build='20210906', buildNumber=20210906, revision=df84a67, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'when_jobWithSameNameManyTimes_then_queryResultSortedBySubmission(com.hazelcast.jet.core.JobTest):'
17:57:15, accumulated pauses: 187 ms, max pause: 126 ms, pauses over 1000 ms: 0
```

--- a/hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java
+++ b/hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java
@@ -559,8 +559,12 @@ public class JobCoordinationService {
                     }
 
                     jobs.entrySet().stream()
-                            .sorted(comparing(Entry<Long, Long>::getValue).reversed())
-                            .forEach(entry -> result.add(tuple2(entry.getKey(), false)));
+                        .sorted(
+                                comparing(Entry<Long, Long>::getValue)
+                                        .thenComparing(Entry::getKey)
+                                        .reversed()
+                        )
+                        .forEach(entry -> result.add(tuple2(entry.getKey(), false)));
                 } else {
                     for (Long jobId : jobRepository.getAllJobIds()) {
                         result.add(tuple2(jobId, false));

